tests/analyse/test_scenario.py

```diff
@@ -846,6 +847,62 @@ def test_model_modelled_obs_ch4(model_scenario_ch4_dummy, footprint_dummy, flux_
     modelled_mf = combined_dataset["mf_mod"].values
     assert np.allclose(modelled_mf, expected_modelled_mf)
 
+@pytest.fixture
+def obs_dummy():
```

Comment:
I don't think this needs to be a fixture if it's only used once.

Reply:
done